### PR TITLE
Syse 288/disable v1 pipeline required check

### PR DESCRIPTION
**Pull request by ermirizio**
Remove v1 api tests pipeline completely

### CODE REVIEW

**Comment by buger** [comment]
**API tests result - postgres15-sha256 env: success** :white_check_mark: 
Branch used: refs/pull/5885/merge
Commit:  
Triggered by: pull_request (@ermirizio)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/7252206664)

**Comment by buger** [comment]
**API tests result - postgres15-murmur64 env: success** :white_check_mark: 
Branch used: refs/pull/5885/merge
Commit:  
Triggered by: pull_request (@ermirizio)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/7252206664)

**Comment by buger** [comment]
**API tests result - mongo44-murmur64 env: success** :white_check_mark: 
Branch used: refs/pull/5885/merge
Commit:  
Triggered by: pull_request (@ermirizio)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/7252206664)

**Comment by buger** [comment]
**API tests result - mongo44-sha256 env: success** :white_check_mark: 
Branch used: refs/pull/5885/merge
Commit:  
Triggered by: pull_request (@ermirizio)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/7252206664)

File: repo-policy/main.tf [hunk 1]
```diff
@@ -49,7 +49,7 @@ module "tyk" {
     { branch    = "master",
       reviewers = "1",
       convos    = "false",
-    required_tests = ["Go 1.21.x Redis 5", "1.21-bullseye", "test (1.21.x, ubuntu-latest, amd64, 15.x)", "api-tests (sha256, mongo44)", "api-tests (sha256, postgres15)", "api-tests (murmur64, mongo44)", "api-tests (murmur64, postgres15)"] },
```
Comment (alephnull) [inline repo-policy/main.tf:52]: Remove 1.21-bullseye

**Review by alephnull — APPROVED** [review]

**Pull request merged**